### ISSUE TITLE
mvnrepository-project isn't published!

### ISSUE THREAD

**Issue by jpink** [devcsrj/mvnrepository-client#1]
Hi!

Your client depend on parent artifact `mvnrepository-project` which isn't published on central maven repository.

Following problem arises when I try to use your client as a dependency on Gradle project:
```
Could not find com.github.devcsrj:mvnrepository-project:1.0.0.
Searched in the following locations:
  - https://jcenter.bintray.com/com/github/devcsrj/mvnrepository-project/1.0.0/mvnrepository-project-1.0.0.pom
  - https://jcenter.bintray.com/com/github/devcsrj/mvnrepository-project/1.0.0/mvnrepository-project-1.0.0.jar
```

**Comment by devcsrj** [comment]
Hi @jpink !

You should instead be including:

```gradle
implementation 'com.github.devcsrj:mvnrepository-api-client:1.0.0'
```

The published artifact is `mvnrepository-api-client`, not the aggregator `mvnrepository-project`. Hope this helps!

---
[Related artifact page](https://search.maven.org/artifact/com.github.devcsrj/mvnrepository-api-client/1.0.0/jar)

**Comment by devcsrj** [comment]
Hi @jpink !

I misunderstood the issue at first. I released `1.0.1` just now to address this. Both the `-project` and `-api-client` are now available, [see nexus](https://oss.sonatype.org/#nexus-search;quick~mvnrepository).

Thanks for catching that!

**Comment by jpink** [comment]
Hi @devcsrj 

sorry, but when i look your nexus link I only see: `mvnrepository-api-client:1.0.0` and "renamed" `mvnrepository-cli:1.0.1`.

I still got error in Gradle build error by using `dependencies { implementation 'com.github.devcsrj:mvnrepository-cli:1.0.1' }`

> Could not find com.github.devcsrj:mvnrepository-project:1.0.1.
>Searched in the following locations:
>  - https://jcenter.bintray.com/com/github/devcsrj/mvnrepository-project/1.0.1/mvnrepository-project-1.0.1.pom
>  - https://jcenter.bintray.com/com/github/devcsrj/mvnrepository-project/1.0.1/mvnrepository-project-1.0.1.jar

You have to publish two missing one, all three artifacts or maybe merge them to single one.

**Comment by devcsrj** [comment]
Hi @jpink 

Yikes. I could've sworn it was there the when I published. 😓 

Alright, I released `1.1.0`, but this time I purged the `cli` module, and flattened the entire thing. The [nexus link](https://oss.sonatype.org/#nexus-search;quick~mvnrepository) should now reflect this:

<img width="605" alt="Screen Shot 2019-06-18 at 9 52 32 PM" src="https://user-images.githubusercontent.com/3963900/59690228-a1513b00-9213-11e9-8431-545b881da80c.png">

Might take a while for indexing sites like [search.maven.org](https://search.maven.org/search?q=mvnrepository) or [mvnrepository](https://mvnrepository.com/artifact/com.github.devcsrj/mvnrepository-api-client) to get updated.

**Comment by jpink** [comment]
Just to verify that now it works with all dependencies. Thanks a lot!
